server/controllers/inventory/reports/prices.js

```diff
@@ -23,7 +23,10 @@ const TEMPLATE = './server/controllers/inventory/reports/prices.handlebars';
 function prices(req, res, next) {
   const params = req.query.params ? JSON.parse(req.query.params) : {};
 
-  if (params && params.group_uuid) { params.group_uuid = db.bid(params.group_uuid); }
+  if (params && params.group_uuid) {
+    db.convert(params)
+    params.group_uuid = db.bid(params.group_uuid);
```

Comment:
This operation is doing that : 

1. HUID --> BUID
2. BUID --> HUID

So why?

Reply:
Whoops, that is a typo.  I'll fix it.  Good catch :+1: